src/main/java/com/parzivail/swg/registry/BlockRegister.java

```diff
@@ -88,10 +89,15 @@ public static void register()
 
 		register(new PBlock("labWall"));
 
-		register(oreChromium = new PBlock("chromium"));
+		register(oreChromium = new PBlock("chromium")); // XXX(kaeptmblaubaer1000): ores should be renamed (with auto-migration to new name) to match ore dictionary names
+								// Then it would be possible to get ores automatically registered in the ore dictionary
```

Comment:
Simply changing the item/block IDs along with the texture names would suffice, we can just classify it as a non-backwards-compatible change, which isn't an issue because we're in Alpha and nobody is playing a real world yet

Reply:
Ok, but does this fit in this PR?

Reply:
It'll be fine, I believe this classifies under the "conforming to the ore dictionary standard" concept